### PR TITLE
REGD-183-gtex-color

### PR DESCRIPTION
**Pull request by mingjiecn**
What I did in this ticket:
1. use gtex color to color sparkline chart, accessibility bar chart and qtl bar chart
2. refactor the code use component BarChart for both accessibility bar chart and qtl bar chart

### CODE REVIEW

File: components/bar-chart.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        organs[0] && GtexColor[organs[0]]
+          ? GtexColor[organs[0]].hex
+          : "#808080";
+      const organsScore = organs.map((organ) =>
+        GtexColor[organ] ? GtexColor[organ].priority : 99
+      );
+      const organsColor = organs.map((organ) =>
+        GtexColor[organ] ? GtexColor[organ].hex : "#808080"
+      );
+      // We elimited all the cases that has duplicated lowest priority.
```
Comment (forresttanaka) [inline components/bar-chart.js]: “eliminated”

File: components/bar-chart.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+        const organsColorSub = organsColor.slice(0, lastIndexPriority + 1);
+        if (organsColorSub.some((x) => x !== firstColor)) {
+          console.log(
+            biosample,
+            ":",
+            organs.join(", "),
+            organsScore.join(", "),
+            organsColor.join(", ")
+          );
+        }
```
Comment (forresttanaka) [inline components/bar-chart.js]: Is this console.log supposed to be there?

**Review by forresttanaka — APPROVED** [review]
Just one thing I wasn’t sure about.

File: components/bar-chart.js (hunk 2 above)
Comment (mingjiecn) [reply components/bar-chart.js]: I wanted to leave the console log here. But maybe it is not a good idea. Let me remove it. Thank you!

**Pull request merged**